integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -339,20 +333,19 @@ func TestProcessesJoinGame100PlayersMultipleTimesRewardAndEndgameInMultipleRound
 		nrRoundsToPropagateMultiShard = 1
 	}
 
+	deployScTx(nodes, idxProposer, string(scCode))
+	time.Sleep(stepDelay)
 	for i := 0; i < nrRoundsToPropagateMultiShard; i++ {
-		time.Sleep(time.Second)
-		deployScTx(nodes, idxProposer, string(scCode))
-		time.Sleep(time.Second)
 		proposeBlock(nodes, idxProposers, round)
-		time.Sleep(time.Second)
```

Comment:
This sleep is required as to be able to correctly sync the header + body

Reply:
propose block has a time.sleep inside